src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,51 @@
+package duke;
+
+import java.time.format.DateTimeParseException;
+
+import duke.exceptions.DukeException;
+import duke.exceptions.InvalidCommandException;
+import duke.exceptions.InvalidTaskNumberException;
+import duke.exceptions.MissingDeadlineException;
+import duke.exceptions.MissingEventTimeException;
+import duke.exceptions.MissingTaskNameException;
+import duke.exceptions.MissingTaskNumberException;
+
+/**
+ * Duke is a Personal Assistant Chatbot that keeps track of various tasks
+ *
+ * @author Adam Oh Zhi Hong
+ */
```

Comment:
Perhaps you can add the `@version` into your class comment, to tell you at which stage the class was last updated.

Reply:
That could be useful, will do that next update!